dxr/app.py

```diff
@@ -211,12 +213,37 @@ def raw(tree, path):
 @dxr_blueprint.route('/<tree>/source/')
 @dxr_blueprint.route('/<tree>/source/<path:path>')
 def browse(tree, path=''):
-    """Show a directory listing or a single file from one of the trees."""
+    """Show a directory listing or a single file from one of the trees.
```

Comment:
Take a look at https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings. Here, for example, the first sentence is the one-line summary. Then we should leave a blank line before describing the exceptional case about NotFound. One thing we do that isn't in PEP 257 is to leave a blank line before the closing quotes of a multi-line docstring. I'm not sure where we got that, but I suspect it was in an older version of PEP 257.